.github/workflows/backend-libraries/release-generic.yml

```diff
@@ -0,0 +1,91 @@
+name: Release
+
+on:
+  workflow_call:
+    inputs:
+      githubappId:
+        required: true
+        type: string
+      versionType:
```

Comment:
You could add the same check on versionType as you did in the frontend-libraries